makefile

```diff
@@ -445,9 +417,9 @@ $(buildDir)/output.%.coverage:$(tmpDir) .FORCE
 	@-[ -f $@ ] && go tool cover -func=$@ | sed 's%$(projectPath)/%%' | column -t
 #  targets to generate gotest output from the linter.
 $(buildDir)/output.%.lint:$(buildDir)/run-linter $(testSrcFiles) .FORCE
-	@./$< --output=$@ --lintArgs='$(lintArgs)' --packages='$*'
+	@./$< --output=$@ --lintBin="$(buildDir)/golangci-lint" --customLinters="$(gopath)/bin/evg-lint -set_exit_status" --packages='$*'
```

Comment:
These 2 targets should have a dependency on $(buildDir)/golangci-lint shouldn't they?